src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts

```diff
@@ -119,15 +119,15 @@ export class TaskListFilterComponent implements OnInit, OnDestroy {
   }
 
   public ngOnDestroy(): void {
-    if (this.subscription && !this.subscription.closed) {
+    if (this.subscription) {
```

Comment:
Query - why have we changed this? Think this was a change by Phil to help with testing or functionality. Is it causing issues or is this a merge conflict?

Reply:
Sure. Merge error I guess